x/marker/types/msg.go

```diff
@@ -470,11 +478,16 @@ func (msg MsgGrantAllowanceRequest) GetSigners() []sdk.AccAddress {
 }
 
 func NewMsgAddFinalizeActivateMarkerRequest(
-	denom string, totalSupply sdkmath.Int,
-	fromAddress, manager sdk.AccAddress,
+	denom string,
+	totalSupply sdkmath.Int,
+	fromAddress sdk.AccAddress,
+	manager sdk.AccAddress,
 	markerType MarkerType,
-	supplyFixed, allowGovernanceControl, allowForcedTransfer bool,
+	supplyFixed bool,
+	allowGovernanceControl bool,
 	accessGrants []AccessGrant,
+	allowForcedTransfer bool,
+	requiredAttributes []string,
```

Comment:
Please put the `accessGrants` argument last. That way, it's the same signature as `NewMsgAddMarkerRequest` with the addition of the `accessGrants` argument.

```suggestion
	supplyFixed bool,
	allowGovernanceControl bool,
	allowForcedTransfer bool,
	requiredAttributes []string,
	accessGrants []AccessGrant,
```

There's some calls to this func that'll need updating too.